examples/discounted-cartrade/README.md

```diff
@@ -86,7 +86,7 @@ Alice will use credentials and other Indy formats such as schema and definition
     - (NOTICE: Before executing the above, your account needs to be added to the docker group (`usermod -a -G docker YourAccount` from root user))
     - On success, this should start two containers:
```

Comment:
@outSH I think "two containers" should be "three containers" here

Reply:
@izuru0 Thanks, done ;)